File: Programing_Basics_with_JS/conditionalStatementsAdvancedMoreExercises/04.CarToGo.js
function carToGo(input) {
    let budget = Number(input[0]);
    let season = input[1];

    let classType = '';
    let carType = '';
    let carPrice = 0;

    if (budget <= 100) {
        classType = 'Economy class';
        if (season === 'Summer') {
            carType = 'Cabrio';
            carPrice = budget * 0.35;
        } else {
            carType = 'Jeep';
            carPrice = budget * 0.65;
        }
    } else if (budget <= 500) {
        classType = 'Compact class';
        if (season === 'Summer') {
            carType = 'Cabrio';
            carPrice = budget * 0.45;
        } else {
            carType = 'Jeep';
            carPrice = budget * 0.80;
        }
    } else {
        classType = 'Luxury class';
        carType = 'Jeep';
        carPrice = budget * 0.9;
    }

    console.log(`${classType}`);
    console.log(`${carType} - ${carPrice.toFixed(2)}`);
}

carToGo(['450', 'Summer'])
carToGo(['450', 'Winter'])
carToGo(['1010', 'Summer'])
carToGo(['1010', 'Winter'])
carToGo(['99.99', 'Summer'])
carToGo(['70.50', 'Winter'])

File: Programming_Basics_with_Python/conditional_statements_advanced_more_exercises/04._car_to_go.py
budget = float(input())
season = input()

car_class = ''
car_type = ''
car_price = 0

if budget <= 100:
    car_class = 'Economy class'
    if season == 'Summer':
        car_type = 'Cabrio'
        car_price = budget * 0.35
    elif season == 'Winter':
        car_type = 'Jeep'
        car_price = budget * 0.65

elif 100 < budget <= 500:
    car_class = 'Compact class'
    if season == 'Summer':
        car_type = 'Cabrio'
        car_price = budget * 0.45
    elif season == 'Winter':
        car_type = 'Jeep'
        car_price = budget * 0.80

elif budget > 500:
    car_class = 'Luxury class'
    car_type = 'Jeep'
    car_price = budget * 0.90

print(f'{car_class} \n{car_type} - {car_price:.2f}')